features/apps_are_up.feature

```diff
@@ -25,7 +25,7 @@ Feature: Apps are up
   @supplier-frontend
   Scenario: Check the supplier frontend is up
     Given I have a URL for "dm_frontend"
-    When I send a GET request to "/suppliers/login"
+    When I send a GET request to "/suppliers"
```

Comment:
Hmm... this is a bit odd, as `/suppliers` will be a redirect (302) to the buyer app login page.  So I'm not sure why this test is passing :confused: - it might be better to point this to a page that doesn't require login though, e.g. `/suppliers/create`